libs/sdk-ui-kit/src/AppearancePage/StylingFooter.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IStylingFooterProps {
+    content?: JSX.Element;
+    buttons?: JSX.Element;
+}
+
+/**
+ * @internal
+ */
+export const StylingFooter = (props: IStylingFooterProps): JSX.Element => {
```

Comment:
Please use `React.FC` type. Can you update all the component in the new packages related to styling and settings widget?